tests/unit/tf/blocks/test_mlp.py

```diff
@@ -71,6 +74,42 @@ def test_mlp_block(
         assert mlp.layers[-1].__class__.__name__ == normalization.__class__.__name__
 
 
+@pytest.mark.parametrize("run_eagerly", [True, False])
+def test_mlp_model_with_sequential_features_and_combiner(
+    sequence_testing_data: Dataset, run_eagerly
+):
+    schema = sequence_testing_data.schema
+    target = schema.select_by_tag(Tags.ITEM_ID).column_names[0]
+
+    loader = Loader(sequence_testing_data, batch_size=8, shuffle=False)
+
+    model = ml.Model(
+        ml.InputBlockV2(
+            schema,
+            categorical=ml.Embeddings(
+                schema.select_by_tag(Tags.CATEGORICAL), sequence_combiner="mean"
+            ),
+            continuous=ml.Continuous(post=SequenceAggregator("mean")),
+        ),
+        ml.MLPBlock([32]),
+        ml.CategoricalOutput(
+            schema.select_by_name(target), default_loss="categorical_crossentropy"
+        ),
+    )
+
+    predict_last = ml.SequencePredictLast(schema=schema.select_by_tag(Tags.SEQUENCE), target=target)
+
+    testing_utils.model_test(
+        model, loader, run_eagerly=run_eagerly, reload_model=False, fit_kwargs={"pre": predict_last}
```

Comment:
Can we test with `reload_model=True` to ensure the model can be re-loaded for new training/eval iterations and to check that the input signatures are correctly saved for serving?

Reply:
Sure. I changed `model_test()`  to `reload_model=True` and the test still passes